packages/react-vapor/src/components/optionPicker/OptionPickerSelectors.ts

```diff
@@ -0,0 +1,18 @@
+import {createSelector} from 'reselect';
+import * as _ from 'underscore';
+
+import {IReactVaporState} from '../../ReactVapor';
+import {IOptionPickerState, optionPickerInitialState} from './OptionPickerReducers';
+
+const get = (state: IReactVaporState, props: {id: string}): IOptionPickerState =>
+    _.findWhere(state.optionPickers, {id: props.id}) || optionPickerInitialState;
```

Comment:
I'm not sure about this default value, why?

With this, I won't be able to detect whether my picker actually exist